### DISCUSSION TITLE
Api route

### DISCUSSION THREAD

**Discussion by radhe65gupta** [vercel/next.js#62450] [App Router]
Hey all, I host my next js app on server,my url like www.example.com .
I request my next route api   www.example.com/api/seeticket
And I want to redirect my home page www.example.com. But it redirected to localhost:3000. not my hosted url www.example.com.
Please help me this issue

Thanks

**Comment by ananuness** [comment]
Could you provide the code of your API route to we see what's the problem?

**Reply by radhe65gupta** [reply]
Thanks @ananuness for responding,
My code is 
import { NextResponse } from "next/server"; //[www.example.com/api/seeticket](https://www.example.com/api/seeticket) export async function GET(request) { const redirectUrl = request.nextUrl.clone(); redirectUrl.pathname = '/'; return NextResponse.redirect(request.nextUrl, 307) }



Any additional confirmation is required in next.config.js file ?
